bin/newDesigner.sh

```diff
@@ -14,6 +14,17 @@ if [ ! -z "$2" ]
     JOB=$2
 fi
 
+unameOut="$(uname -s)"
```

Comment:
please come with an another PR for this change... 
i see the mac-dev.sh as completed - but this has not much to do with the other changes

Reply:
what I'd like to change here: can you isolate that code in a function and only execute it if .env template has no value set in this variable? or even remove it completely from .env (if we can determine the correct version also for windows)

Reply:
sure will do :)